app/src/main/res/values-pt/readers.xml

```diff
@@ -0,0 +1,223 @@
+        <item>http://download.quranicaudio.com/quran/muhammad_siddeeq_al-minshaawee/</item>
+        <item>http://download.quranicaudio.com/quran/mahmood_khaleel_al-husaree/</item>
+        <item>http://download.quranicaudio.com/quran/abdurrahmaan_as-sudays/</item>
+        <item>http://download.quranicaudio.com/quran/nasser_bin_ali_alqatami/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Abdul_Basit_Murattal_192kbps/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Abdul_Basit_Mujawwad_128kbps/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Abdullah_Basfar_192kbps/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Abdurrahmaan_As-Sudais_192kbps/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Abu_Bakr_Ash-Shaatree_128kbps/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Alafasy_128kbps/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Ghamadi_40kbps/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Ibrahim_Walk_192kbps_TEST/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Hani_Rifai_192kbps/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Husary_128kbps_Mujawwad/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Hudhaify_128kbps/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Maher_AlMuaiqly_64kbps/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Minshawy_Mujawwad_192kbps/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Mohammad_al_Tablaway_128kbps/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Muhammad_Ayyoub_128kbps/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Muhammad_Jibreel_128kbps/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Saood_ash-Shuraym_128kbps/</item>
+        <item>http://mirrors.quranicaudio.com/everyayah/Yasser_Ad-Dussary_128kbps/</item>
+        <item>http://download.quranicaudio.com/quran/abdul_basit_murattal/</item>
+        <item>http://download.quranicaudio.com/quran/abdulbaset_mujawwad/</item>
+        <item>http://download.quranicaudio.com/quran/aziz_alili/</item>
+        <item>http://download.quranicaudio.com/quran/salahbudair/</item>
+        <item>http://download.quranicaudio.com/quran/sa3ood_al-shuraym/</item>
+        <item>http://download.quranicaudio.com/quran/yasser_ad-dussary/</item>
+        <item>http://download.quranicaudio.com/quran/mohammad_altablawi/</item>
+        <item>http://download.quranicaudio.com/quran/sahl_yaaseen/</item>
+        <item>http://download.quranicaudio.com/quran/abu_bakr_ash-shaatree/</item>
+        <item>http://download.quranicaudio.com/quran/ahmad_nauina/</item>
+        <item>http://download.quranicaudio.com/quran/akram_al_alaqmi/</item>
+        <item>http://download.quranicaudio.com/quran/ali_hajjaj_alsouasi/</item>
+    </string-array>
+
+    <!-- actual qari names -->
```

Comment:
all you need is lines 1, 2, and 188 to the end. all the rest should not be overwritten since they'll automatically be pulled from the english file.